src/visit_vtk/lightweight/vtkVisItCellLocator.h

```diff
@@ -293,8 +293,8 @@ class VISIT_VTK_LIGHT_API vtkVisItCellLocator : public vtkLocator
 
 
 private:
-  vtkVisItCellLocator(const vtkVisItCellLocator&);  // Not implemented.
-  void operator=(const vtkVisItCellLocator&);  // Not implemented.
+  vtkVisItCellLocator(const vtkVisItCellLocator&) = delete;
+  void operator=(const vtkVisItCellLocator&) = delete;
```

Comment:
if this isn't an issue then batch resolve all the comments, but if it is I thought it'd be good to mark all of these with a comment.